src/handlers/ReactNativeUnifiedPlan.ts

```diff
@@ -393,7 +393,7 @@ export class ReactNativeUnifiedPlan extends HandlerInterface
 		await this._pc.setLocalDescription(offer);
 
 		// We can now get the transceiver.mid.
-		const localId = transceiver.mid;
+		let localId = transceiver.mid;
 
 		// Set MID.
 		sendingRtpParameters.mid = localId;
```

Comment:
This means that `sendingRtpParameters.mid` is `undefined`. And later in line 460, when we call `this._remoteSdp!.send()` we are passing `sendingRtpParameters` without `mid`, so the remote SDP section is being constructed **without** MID. I'm worried about the implications of this.

Reply:
Not true since we read the MID from the parser local media section, which does contain `a=mid`.